cluster/src/main/java/com/hartwig/pipeline/calling/SubStageInputOutput.java

```diff
@@ -15,9 +18,13 @@
     OutputFile outputFile();
 
     @Value.Parameter
-    BashStartupScript currentBash();
+    List<BashCommand> bash();
+
+    static SubStageInputOutput of(final String sampleName, final OutputFile outputFile, final List<BashCommand> bash) {
+        return ImmutableSubStageInputOutput.of(sampleName, outputFile, bash);
+    }
 
-    static SubStageInputOutput of(final String sampleName, final OutputFile outputFile, final BashStartupScript currentBash) {
-        return ImmutableSubStageInputOutput.of(sampleName, outputFile, currentBash);
+    static SubStageInputOutput seed(final String sampleName) {
```

Comment:
i guess the idea was that this would "seed" a sub-pipeline with an empty i/o object. if it confuses you probably confuses most :) I'll change it to empty()